parsigs/parse_sig_api.py

```diff
@@ -61,6 +67,16 @@ class StructuredSig:
 inflect_engine = inflect.engine()
 
 
+def _open_latin_type_dict():  # add parameters path, name
```

Comment:
@yoad-hordan4 this file needs to be in the `resources` folder
To get its path after for opening, look at the create spell checker `def`
`Path(__file__).parent / 'resources/$FILE_NAME'`